eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/json/JsonReaders.scala

```diff
@@ -79,12 +80,74 @@ object JsonReaders {
     }
   }
 
+  implicit val shortChannelIdReads: Reads[ShortChannelId] = {
+    Reads { jsValue =>
+      SerializerUtil.processJsString(ShortChannelId.fromHex)(jsValue)
+    }
+  }
+
   implicit val channelDescReads: Reads[ChannelDesc] = {
     Json.reads[ChannelDesc]
   }
 
-  implicit val channelInfoReads: Reads[ChannelInfo] = {
-    Json.reads[ChannelInfo]
+  implicit val normalChannelStateReads: Reads[ChannelState.NORMAL.type] =
```

Comment:
Maybe we should just make a `Reads[ChannelState]` and then use the `fromString` method in the `ChannelState` companion object to serialize.

Reply:
That's already in the file further up, this is for validating a specific value